README.md

```diff
@@ -425,7 +425,7 @@ Password of postgres user `AUTH_SERVICE_PG_USER`.
 
 ##### `AUTH_SERVICE_PG_SSL_ENABLED` [`false`]
 
-Whether an SSL connection shall be used to connect to postgres.
+Whether an SSL connection shall be used to connect to postgres. If `true`, then `AUTH_SERVICE_PG_CA_CERT` (probably) must be set to a valid value (see nuance about override in description of this variable).
```

Comment:
Is this `nuance about override in description` referring to the Joi description? I'd rather see a copy of that information here in the readme than have to dive into the code to find it.

Reply:
Tiny clarification added :)

Resolving.